test/metric_learn_test.py

```diff
@@ -149,5 +149,48 @@ def test_iris(self):
     self.assertLess(csep, 0.25)
 
 
+class TestPGDM(MetricTestCase):
+  def test_iris(self):
+
+    # Generate full set of constraints for comparison with reference implementation
+    n = self.iris_points.shape[0]
+    a, b, c, d = [], [], [], []
+    for i in range(n):
+      for j in range(i+1, n):
+        if self.iris_labels[i] == self.iris_labels[j]:
+          a.append(i)
+          b.append(j)
+        else:
+          c.append(i)
+          d.append(j)
+
+    # Full metric
+    pgdm = PGDM(convergence_threshold = 0.01)
```

Comment:
Style nitpick: don't leave spaces around the `=` in keyword arguments. (here and throughout)